android/app/src/main/java/com/chorestory/app/MainActivity.java

```diff
@@ -16,6 +18,12 @@ protected void onCreate(Bundle savedInstanceState) {
         App.getAppComponent().inject(this);
         setContentView(R.layout.activity_main);
 
+        // Register Notification Channel
+        NotificationService notificationService = new NotificationService();
+        notificationService.createNotificationChannel(getBaseContext(), getString(R.string.notification_channel_id));
```

Comment:
will notifications work with phones running API 25 or less?

Reply:
yes, CHANNEL_ID is only needed for >= 26 and is ignored for  <= 25